unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/Environment.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace DCL
+{
+    public class Environment : Singleton<Environment>
+    {
+        public MessagingControllersManager messagingControllersManager { get; }
+
+        /*
+         * TODO: Continue moving static beans to this class
+         * 
+        MemoryManager memoryManager;
+        PointerEventsController pointerEventsController;
+        ParcelScenesCleaner parcelScenesCleaner; // This is a static member of ParcelScene
+        PoolManager poolManager; // This should be created through a Factory, and that factopry should execute the code in the method EnsureEntityPool
```

Comment:
We should remove those comments.

Reply:
I would prefer to keep them until we move all the singleton instances to this class. They are just here to remember what we need to do.

Reply:
Fair enough.